### PR TITLE
PAAS-2178: set replication and galera_cluster to false in dd conf for single nodes

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2178

Short description:

### CODE REVIEW

File: mixins/mariadb.yml [hunk 1]
```diff
@@ -107,8 +112,8 @@ actions:
         echo "      - 'env:${env.envName}'" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
         echo "      - 'role:database'" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
         echo "    options:" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
-        echo "       replication: true" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
-        echo "       galera_cluster: true" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
+        echo "       replication: ${this.cluster}" >> /etc/datadog-agent/conf.d/mysql.d/conf.yaml
```
Comment (Ronan-Gorain) [inline mixins/mariadb.yml:115]: This check is always useless event with galera cluster. We don't use mysql replication

**Review by Ronan-Gorain — CHANGES_REQUESTED** [review]

File: mixins/mariadb.yml (hunk 1 above)
Comment (adoux) [reply mixins/mariadb.yml]: Good call

Comment (adoux) [reply mixins/mariadb.yml]: fixed

**Comment by adoux** [comment]
I also added updated the v19 migration package

**Review by Ronan-Gorain — APPROVED** [review]

**Pull request merged**